install_config/monitoring/configuring-etcd-monitoring.adoc

```diff
@@ -31,64 +31,16 @@ prometheus-k8s-1                               0/4       ContainerCreating   0
 prometheus-operator-6c9fddd47f-9jfgk           1/1       Running             0          36m
```

Comment:
It is better change to prometheus-k8s-1 status to Running, example
prometheus-k8s-1                               4/4       Running   1          35m